src/LinuxTracing/PerfEvent.h

```diff
@@ -201,6 +203,8 @@ using SchedSwitchPerfEvent = TypedPerfEvent<SchedSwitchPerfEventData>;
 
 struct SchedWakeupPerfEventData {
   pid_t woken_tid;
+  pid_t was_blocked_by_thread;
+  pid_t was_blocked_by_process;  // Could be zero if we don't have access to the process id.
```

Comment:
I need clarification on "Could be zero if we don't have access to the process id":
- Only the process or both the process and the thread?
- When does this happen?